questionnaire/form-helper.js

```diff
@@ -96,6 +97,7 @@ function renderSection({
                                 }) }}
                             {% endif %}
                         <input type="hidden" name="_csrf" value="${csrfToken}">
+                        <input type="hidden" name="questionnaireId" value="${questionnaireId}">
```

Comment:
For consistency, we should probably use an underscore prefix on this too e.g. none of our user requested data can start with an underscore. Doesn't mean the in code variable should be prefixed though, just the html